.gitignore

```diff
@@ -0,0 +1,5 @@
+# Dependencies
```

Comment:
Can we add these into gitignore too? https://github.com/CS4215-GN-IY/c-slang/blob/gn/try-antlr/.gitignore